lib/dateParser-lambda.js

```diff
@@ -0,0 +1,146 @@
+const AWS = require('aws-sdk')
+AWS.config.region = 'us-east-1'
+const lambda = new AWS.Lambda()
+
+const parseDate = async function (date) {
+  const classifications = classifyDate(date)
+  let preparsedDate = preparse(date, classifications)
+  let dateRange
+  try {
+    dateRange = await timeTwist(preparsedDate)
+  } catch (e) {
+    console.error(e)
+  }
+  // If timetwister failed to meaningfully do anything, hail mary to at least have a range of years
+  if (Object.values(dateRange).includes(null)) {
+    preparsedDate = parseOnlyYears(preparsedDate)
+  }
+  return dateRange.map((range) => [range.start, range.end])
+}
+
+const timeTwist = async (preparsedDate) => {
+  const payloadStr = JSON.stringify({
+    path: '/',
+    body: JSON.stringify({ dates: [preparsedDate] })
```

Comment:
Note that the date-parser does by design support parsing multiple dates at a time. We should find a way to send all of the dates we want to process in a given batch in single call if possible - to reduce processing time / invocation cost